documentation/asciidoc/accessories/camera/qt.adoc

```diff
@@ -0,0 +1,22 @@
+=== Using _libcamera_ and _Qt_ together
+
+_Qt_ is a popular application framework and GUI toolkit, and indeed _libcamera-apps_ optionally makes use of it to implement a camera preview window.
+
+However, _Qt_ defines certain symbols as macros in the global namespace (such as `slot` and `emit`) and this causes errors when including _libcamera_ files. The problem is common to all platforms trying to use both _Qt_ and _libcamera_ and not specific to Raspberry Pi. Nonetheless we suggest that developers experiencing difficulties try the following workarounds.
```

Comment:
@aallan Minor nitpick, but should there be a colon instead of a full-stop at the end of this line?